components/notifier/tests/fixtures.py

```diff
@@ -0,0 +1,73 @@
+"""Fixtures for tests."""
+
+from typing import cast
+
+from shared.utils.type import (MetricId, NotificationDestinationId, SourceId,
+                               SubjectId)
+
+METRIC_ID = cast(MetricId, "metric_uuid")
+METRIC_ID2 = cast(MetricId, "metric_uuid2")
+NOTIFICATION_DESTINATION_ID = cast(
+    NotificationDestinationId, "destination_uuid")
+SOURCE_ID = cast(SourceId, "source_uuid")
+SUBJECT_ID = cast(SubjectId, "subject_uuid")
+
+
+def create_report(
+        title: str = "Title", report_uuid: str = "report1",
+        last: bool = True, deleted=None, metric_id: MetricId = METRIC_ID) -> dict:
+    """returns a fake report
+
+    Args:
+        title (str, optional): Title of the report. Defaults to "Title".
+        report_uuid (str, optional): Fake UUID. Defaults to "report1".
+        last (bool, optional): last report indication. Defaults to True.
+        deleted (_type_, optional): deletion indication. Defaults to None.
+        metric_id (MetricId, optional): fake metric id. Defaults to METRIC_ID.
+
+    Returns:
+        dict: Report
+    """
+    report = dict(
+        report_uuid=report_uuid,
+        title=title,
+        subjects={
+            SUBJECT_ID: dict(
+                name="Subject",
+                type="subject_type",
+                metrics={
+                    metric_id: dict(
+                        name="Metric",
+                        type="violations",
+                        addition="sum",
+                        target="0",
+                        accept_debt=False,
+                        tags=["security"],
+                        scales=["count", "percentage"],
+                        sources={
+                            SOURCE_ID: dict(
+                                type="sonarqube",
+                                name="Source",
+                                parameters=dict(
+                                    url="https://url", password="password"),
+                            )
+                        },
+                    )
+                },
+            )
+        },
+        notification_destinations={
+            NOTIFICATION_DESTINATION_ID: dict(
+                teams_webhook="", name="notification_destination", url="https://reporturl"
+            )
+        },
+        # Sonarcloud says this is a security hotspot, but it's not.
+        issue_tracker=dict(type="jira", parameters=dict(
+            username="jadoe", password="secret")),  # NOSONAR
```

Comment:
This can be suppressed in SonarCloud. No need to clutter the code with it:
```suggestion
        issue_tracker=dict(type="jira", parameters=dict(
            username="jadoe", password="secret")),
```